src/NJsonSchema.CodeGeneration.CSharp/Templates/Enum.liquid

```diff
@@ -5,10 +5,11 @@
 {{ TypeAccessModifier }} enum {{ Name }}
 {
 {% for enum in Enums -%}
+    [System.ComponentModel.Description("{{ enum.Description }}")]
```

Comment:
Only render attribute if description is available...

Reply:
Not familiar with this liquid format, how does one do that?